content/22.vaccines-trad.md

```diff
@@ -458,6 +459,22 @@ In general, deciding on the prioritization and allocation of the COVID-19 vaccin
 Various frameworks, models, and methods have been proposed to tackle these issues with many countries, regions, or U.S. states devising their own distribution and administration plans [@doi:10.1126/science.abe6959; @doi:10.1038/s41598-020-78447-3; @doi:10.1016/j.vaccine.2020.12.059; @doi:10.1136/medethics-2020-107036; @doi:10.1101/2021.01.12.21249694].
 The majority of the distribution plans prioritized offering vaccines to key workers such as health care workers and those who are clinically vulnerable, such as the elderly, the immunocompromised, and individuals with comorbidities, before targeting the rest of the population, who are less likely to experience severe outcomes from COVID-19 [@url:https://ourworldindata.org/covid-vaccination-policy].
 The availability of vaccines developed in a variety of countries using a variety of platforms is likely to work in favor of worldwide vaccine access.
+Hence why the development of Corbevax, a COVID-19 protein subunit vaccine developed by Texas Children’s Hospital and Baylor College of Medicine is an important initiaitve as it is patent free and so its manufactuing specifications can be shared globally. 
+Corbevax can be produced at low-cost using existing techonology that is now licensed to Biological E. Limited (BioE) for production and is approved for distribution in India and Botswana [@url:https://theconversation.com/corbevax-a-new-patent-free-covid-19-vaccine-could-be-a-pandemic-game-changer-globally-174672; @url:https://covid19.trackvaccines.org/vaccines/54/].
```

Comment:
```suggestion
Corbevax can be produced at low cost using existing techonology and is now licensed to Biological E. Limited (BioE), an Indian company specializing in low-cost vaccine production [@url:https://theconversation.com/corbevax-a-new-patent-free-covid-19-vaccine-could-be-a-pandemic-game-changer-globally-174672].
The vaccine has been approved for distribution in India and Botswana [@url:https://covid19.trackvaccines.org/vaccines/54].
```

Reply:
We may end up moving some of this text since reviewer 1 wanted us to add more vaccines, and this seems like a good one to add! But works great here now.

Reply:
Sounds good